pkg/plugin/util/util.go

```diff
@@ -284,25 +286,31 @@ func IsObjectBlocked(item runtime.Unstructured) (bool, string, error) {
 	if err != nil {
 		return false, "", errors.Errorf("Could not translate item kind %s to CRD name", item.GetObjectKind())
 	}
-	if IsResourceBlocked(crdName) {
+
+	blockListConfigMap, err := cmd.RetrieveBlockListConfigMap()
+	if err != nil {
+		return false, "", errors.WithStack(err)
+	}
+	if IsResourceBlocked(crdName, blockListConfigMap) {
 		return true, crdName, nil
 	}
 	return false, crdName, nil
 }
 
-func GetResources() []string {
-	desiredResources := make([]string, len(constants.ResourcesToHandle)+len(constants.ResourcesToBlock))
+func GetResources(blockListConfigMap map[string]string) []string {
+	desiredResources := make([]string, len(constants.ResourcesToHandle)+len(blockListConfigMap))
 	for resourceToHandle, _ := range constants.ResourcesToHandle {
 		desiredResources = append(desiredResources, resourceToHandle)
 	}
-	for resourceToBlock, _ := range constants.ResourcesToBlock {
+	for resourceToBlock, _ := range blockListConfigMap {
 		desiredResources = append(desiredResources, resourceToBlock)
 	}
 	return desiredResources
 }
 
-func IsResourceBlocked(resourceName string) bool {
-	return constants.ResourcesToBlock[resourceName]
+func IsResourceBlocked(resourceName string, resourcesToBlock map[string]string) bool {
+	isBlocked, _ := strconv.ParseBool(resourcesToBlock[resourceName])
```

Comment:
`resourcesToBlock` could be nil

Reply:
Added unit test for nil case. Golang can handle this.